app/models/message.rb

```diff
@@ -0,0 +1,11 @@
+class Message < ActiveRecord::Base
+
+  belongs_to :sender, class_name: 'User'
+  belongs_to :recipient, class_name: 'User'
+
+  validates_presence_of :body, :sender_id, :recipient_id
+
+  scope :between, -> (sender_id, recipient_id) do
+    where('(sender_id = ? AND recipient_id = ?) OR (sender_id = ? AND recipient_id = ?)', sender_id, recipient_id, recipient_id, sender_id)
```

Comment:
Line is too long. [139/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.